WcaOnRails/lib/auxiliary_data_computation.rb

```diff
@@ -80,7 +81,6 @@ def self.compute_rank_tables
         personal_records_with_event = ActiveRecord::Base.connection.execute <<-SQL
           SELECT eventId, personId, countryId, continentId, min(#{field}) value
           FROM #{concise_table_name}
-          WHERE eventId <> '333mbo'
```

Comment:
Just to make it clear, '333mbo' has rank 990 (and the highest rank)?

Reply:
Oops I didn't see it was in a different method.
Then this is possible because we don't fill the concise table with mbo results?

Reply:
Yep, in fact it has the rank of `999` https://github.com/thewca/worldcubeassociation.org/blob/e909979644941d6bb1dbdef02769b7a76e074b81/WcaOnRails/db/seeds/events.seeds.rb#L24
As a result it doesn't end up in `Concise*Results` and so we don't need this condition when selecting from those. This particular change didn't seem to impact performance at all, it's just a random cleanup ^^